orders-war/src/main/java/au/gov/qld/pub/orders/entity/Item.java

```diff
@@ -163,8 +168,12 @@ public String getQuantityPaid() {
         Map<String, String> map = JsonHelper.deserialise(Map.class, fields);
         return map != null ? map : new TreeMap<String, String>();
     }
-        
-    public void setFields(Map<String, String> fields) {
+
+    public void setFields(String fields) {
+        this.fields = fields;
+    }
+
+    public void setFieldsFromMap(Map<String, String> fields) {
```

Comment:
renamed helper method used in tests so beanUtils copy properties works in createItem function